webClient/src/app/browser-panel/browser-panel.component.ts

```diff
@@ -56,25 +57,23 @@ class FileRow {
     './browser-panel.component.scss',
     '../../styles.scss'
     ]
+    
 })
-export class BrowserPanelComponent implements OnInit {
+export class BrowserPanelComponent implements AfterViewInit, OnInit {
     // @ts-ignore
     @Input() connection: Connection;
     // @ts-ignore
```

Comment:
Using ts-ignore is generally not a great idea (and it doesn't come from our code but we shouldn't leave this here) 
If you remove them and Typescript gives you compile problems, and the problems are via declaration, you can just do something like

@Input() connection: Connection;
connection; // To ignore TS compile issue

Reply:
@DivergentEuropeans you mean use
connection; instead of @input() connection: Connection;

or having both
@input() connection: Connection;
connection; // To ignore TS compile issue ?